### ISSUE TITLE
segnet error?

### ISSUE THREAD

**Issue by ebeowulf** [chainer/chainercv#487]
I'm trying to run the segnet demo.py example and am getting errors. I've looked at the function files (segnet_basic.py), but am unsure of what the network is supposed to be doing. 

> python demo.py 0001TP_008550.png
Traceback (most recent call last):
  File "demo.py", line 45, in <module>
    main()
  File "demo.py", line 32, in main
    labels = model.predict([img])
  File "/home/ebeowulf/.local/lib/python2.7/site-packages/chainercv/links/model/segnet/segnet_basic.py", line 173, in predict
    score = self.__call__(x)[0].data
  File "/home/ebeowulf/.local/lib/python2.7/site-packages/chainercv/links/model/segnet/segnet_basic.py", line 137, in __call__
    h = _without_cudnn(p1, F.relu(self.conv1_bn(self.conv1(h))))
  File "/home/ebeowulf/.local/lib/python2.7/site-packages/chainercv/links/model/segnet/segnet_basic.py", line 14, in _without_cudnn
    return f(x)
TypeError: 'MaxPooling2D' object is not callable

Note that I've tried both with and without GPU support, and am receiving the same error. I've also freshly installed both chainer and chainercv.

**Comment by yuyu2172** [comment]
Thank you for reporting.
This code has broken due to incompatible change introduced by Chainer 3.0.0.

For the time being, please modify `_without_cudnn` as follows:

```python
def _without_cudnn(f, x):
    with chainer.using_config('use_cudnn', 'never'):
        return f.forward((x,))
```

**Comment by yuyu2172** [comment]
Fixed in #445.
